modules/dnn/test/test_misc.cpp

```diff
@@ -463,6 +463,39 @@ TEST_P(Async, set_and_forward_all)
     }
 }
 
+typedef testing::TestWithParam<tuple<int, Target> > Async;
+TEST_P(Async, forward_two_nets)
```

Comment:
Perhaps, this test should have separate scope due there is no Asynchronous mode is tested.